modules/apps/document-library/document-library-video/src/main/java/com/liferay/document/library/video/internal/preview/DLVideoDLPreviewRendererProvider.java

```diff
@@ -120,15 +123,17 @@ private void _checkForPreviewGenerationExceptions(FileVersion fileVersion)
 		}
 	}
 
+	private static final Snapshot<DLVideoRenderer> _dlVideoRendererSnapshot =
```

Comment:
I do not think that here is needed.
We are on the document-library-video module so if it's failing, this should be an error.

Reply:
Thanks, I wasn't sure on this one since it is in a document library video module. It makes sense that an error should appear instead in this case. I will remove this change.